### ISSUE TITLE
Unable to install typescript@2.3.4 via npm 5: EINVALIDTAGNAME

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#27711]
* have npm v5
* try to npm install

=> 
```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "typescript@2.3.4": Tags may not have any characters that encodeURIComponent encodes.
```

**Comment by octoploid** [comment]
All the npm-shrinkwrap.json files need an update, too.

**Comment by mjbvz** [comment]
Fixed ts specific issue with cc0d24a
